docker/k8s/ehsm-kms.yaml

```diff
@@ -98,6 +98,13 @@ spec:
       - name: couchdb
         image: couchdb:3.2
```

Comment:
Config the CouchDB to run as single node mode in K8S

passed all the unit-test

Signed-off-by: <your intel email>